File: support_tools/swigwin-3.0.12/Examples/test-suite/go/director_enum_runme.go
package main

import "./director_enum"

type MyFoo struct{} // From director_enum.Foo
func (p *MyFoo) Say_hi(val director_enum.EnumDirectorHello) director_enum.EnumDirectorHello {
	return val
}

func main() {
	b := director_enum.NewFoo()
	a := director_enum.NewDirectorFoo(&MyFoo{})

	if a.Say_hi(director_enum.Hello) != b.Say_hello(director_enum.Hi) {
		panic(0)
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/director_enum_runme.py
import director_enum


class MyFoo(director_enum.Foo):

    def say_hi(self, val):
        return val


b = director_enum.Foo()
a = MyFoo()

if a.say_hi(director_enum.hello) != b.say_hello(director_enum.hi):
    raise RuntimeError
